plugin/julia-neo.vim

```diff
@@ -0,0 +1,18 @@
+function! s:RequireJuliaHost(name)
+  " Julia host arguments
+  let args = ['-e', 'using Neovim; start_host()']
+
```

Comment:
I don't really think we should use the same model as python2/python3. The difference is much smaller, at least in practice as much of the ecosystem supports both julia stable and master. (and IMHO it's a matter of preference if stable or master is "the" julia) . Let's just make it configurable for now with `g:julia_host_prog`